src/windows/region.py

```diff
@@ -167,17 +161,11 @@ def __init__(self, file=None, clip=None):
         self.sliderVideo.setMinimum(1)
         self.sliderVideo.setMaximum(self.video_length)
         self.sliderVideo.setSingleStep(1)
-        self.sliderVideo.setSingleStep(1)
         self.sliderVideo.setPageStep(24)
 
-        # Determine if a start or end attribute is in this file
-        start_frame = 1
-        # if 'start' in self.file.data.keys():
-        #     start_frame = (float(self.file.data['start']) * self.fps) + 1
-
         # Display start frame (and then the previous frame)
-        QTimer.singleShot(500, functools.partial(self.sliderVideo.setValue, start_frame + 1))
-        QTimer.singleShot(600, functools.partial(self.sliderVideo.setValue, start_frame))
+        QTimer.singleShot(500, functools.partial(self.sliderVideo.setValue, 2))
```

Comment:
Not sure why we start at frame 2 and then jump to 1... I think this is a hack to correctly trigger the `valueChanged()` signal. But I see no reason to change this at the moment, until we have time to investigate it further.